fdbserver/Status.actor.cpp

```diff
@@ -1944,7 +1945,14 @@ static JsonBuilderObject tlogFetcher(int* logFaultTolerance, const std::vector<T
 			if(currentFaultTolerance >= 0) {
 				localSetsWithNonNegativeFaultTolerance++;
 			}
-			minFaultTolerance = std::min(minFaultTolerance, currentFaultTolerance);
+
+			if (tLogs[i].locality == tagLocalitySatellite) {
+				// FIXME: This hack to bump satellite fault tolerance, is to make it consistent
+				//  with 6.2.
+				minFaultTolerance = std::min(minFaultTolerance, currentFaultTolerance + 1);
```

Comment:
This line seems to be the only functional change. 

In future PRs, it will be better to have a style change PR followed by a real change PR. That makes review easier.

Reply:
This PR has 3 commits which can be reviewed individually fairly easily. I made a single PR as they were somewhat related in scope, but can open different PRs in future.